terraform/modules/k8s/istio/main.tf

```diff
@@ -32,11 +32,12 @@ resource "helm_release" "istio-init" {
   version    = var.istio_version
   namespace  = var.istio_namespace
   repository = "istio"
+  depends_on = [kubernetes_namespace.istio]
 }
 
 resource "null_resource" "delay" {
   provisioner "local-exec" {
-    command = "timeout 200 bash -c 'until $(kubectl get --all-namespaces gateways.networking.istio.io && kubectl get --all-namespaces envoyfilters.networking.istio.io && kubectl get --all-namespaces policies.authentication.istio.io && kubectl get --all-namespaces destinationrules.networking.istio.io && kubectl get --all-namespaces virtualservices.networking.istio.io && kubectl get --all-namespaces envoyfilters.networking.istio.io && kubectl get --all-namespaces attributemanifests.config.istio.io && kubectl get --all-namespaces handlers.config.istio.io && kubectl get --all-namespaces meshpolicies.authentication.istio.io); do sleep 5; done'"
+    command = "timeout 200 bash -c 'until [ $(kubectl get customresourcedefinitions -l release=istio --no-headers | wc -l) -ge 23 ]; do sleep 5; done'"
```

Comment:
Please, extract the `23` value to a local variable

Reply:
Done in 82b149c.